### ISSUE TITLE
User Feedback Form  收集用户反馈

### ISSUE THREAD

**Issue by ecr-oha-lime** [rocksdanister/lively#1297]
推送给使用Lively的用户
Sent to users using Lively

https://forms.office.com/r/MVvLuB0Rxh

**Comment by rocksdanister** [comment]
In app survey is better 🤔

Example: 
![popup1](https://user-images.githubusercontent.com/17554161/178500237-497407c3-aba5-4800-be6b-344553c07f6c.png)
Valve Hardware Survey

**Comment by ecr-oha-lime** [comment]
雀食
但是你怎么加进去？
=======translate=======
indeed
But how do you add it？

**Comment by ecr-oha-lime** [comment]
在下个版本么？
======translate======
In the next version?

**Comment by ecr-oha-lime** [comment]
这是用Microsoft Forms构建的，你可以通过iframe使用。
========translate=========
This is built with Microsoft Forms and you can use it via iframe.

`<iframe width="640px" height="480px" src="https://forms.office.com/Pages/ResponsePage.aspx?id=M09rEm5L206y9osd-wHJnJUSKcnlTOJHnxgcg9W5idtUQkw5QlBDTlZUQkVCRzBHWkgwVk5ZSlVYUi4u&embed=true" frameborder="0" marginwidth="0" marginheight="0" style="border: none; max-width:100%; max-height:100vh" allowfullscreen webkitallowfullscreen mozallowfullscreen msallowfullscreen> </iframe>`

**Comment by ecr-oha-lime** [comment]
@rocksdanister

**Comment by rocksdanister** [comment]
No, ideally the surveys will be custom made in-app using WinUI3 controls.
